udata/core/user/models.py

```diff
@@ -65,6 +65,9 @@ class User(WithMetrics, UserMixin, db.Document):
     # when SECURITY_CONFIRMABLE is True
     confirmed_at = db.DateTimeField()
 
+    password_rotation_demand = db.DateTimeField()
```

Comment:
```suggestion
    password_rotation_demanded = db.DateTimeField()
```

or 

```suggestion
    password_rotation_needed = db.DateTimeField()
```

Reply:
Ideally we should even add a `_at` suffix to indicate it's a date, but that might make the field names a bit long.